src/account/default.ts

```diff
@@ -36,41 +41,52 @@ export class Account extends Provider implements AccountInterface {
    * @param transaction - transaction to be invoked
    * @returns a confirmation of invoking a function on the starknet contract
    */
-  public override async invokeFunction(
-    { contractAddress, calldata = [], nonce, entrypoint, ...invocation }: Invocation,
-    _abi?: Abi
+  public async execute(
```

Comment:
Might want to change the comment on this function above `Invoke a function` -> `call execute function in account contract` also the reference is probably not relevant for this function